docs/user/tutorials/quickstart.md

```diff
@@ -0,0 +1,138 @@
+# Quickstart
+
+This guide covers how you can quickly get started using Hypper.
+
+If you already know Helm, this will be a breeze, as Hypper follows Helm's UI,
```

Comment:
I do not like the term UI here, I'd opt for something like `... as Hypper follows Helm's workflow,...`

Reply:
I reworked this paragraph as Hypper is no longer a drop-in replacement.